assets/docs.html

```diff
@@ -150,33 +147,34 @@ <h2 class="docs-subtitle">Contributors</h2>
 					<div class="span4">
 						<div class="footer-title">ABOUT US</div>
 						WindBot is a high quality automation and enhancement software that functions alongside the Tibia client. It works by sending mouse clicks and keyboard strokes, displaying it's own visuals on top, facilitating player communication and is customizable in many many ways.
-						<br><br>
+						<br /><br />
 						It was created by Lucas Terra, one of the main minds behind the success of NeoBot and ElfBot. It took several months of work and tests before...
-						<br><br>
+						<br /><br />
 						<a href="about-us.html" class="blue">Read more »</a>
 					</div>
 					<div class="span4">
 						<!-- Sections -->
 						<div class="footer-title">SECTIONS</div>
-						<a href="news.html">» News</a><br><br>
-						<a href="features.html">» Features</a><br><br>
-						<a href="download.html">» Download</a><br><br>
-						<a href="purchase.html">» Purchase</a><br><br>
-						<a href="lua-variables.html">» Documentation</a><br><br>
+						<a href="news.html">» News</a><br /><br />
+						<a href="features.html">» Features</a><br /><br />
+						<a href="download.html">» Download</a><br /><br />
+						<a href="purchase.html">» Purchase</a><br /><br />
+						<a href="lua-variables.html">» Documentation</a><br /><br />
```

Comment:
Long story short its dependent on doctype, mostly optional as stated in specs, in practice required for proper syntax highlighting. Anyway it's good to have tags always closing. 

Ultimately it's a matter of code style which we don't have. I'm not the one accepting PR's.